app/src/main/java/com/sun/americanroom/screen/explore/adapter/RoomExploreAdapter.kt

```diff
@@ -0,0 +1,51 @@
+package com.sun.americanroom.screen.explore.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.viewpager.widget.PagerAdapter
+import com.bumptech.glide.Glide
+import com.sun.americanroom.R
+import com.sun.americanroom.data.model.RoomExplore
+
+class RoomExploreAdapter : PagerAdapter() {
+
+    private var context: Context? = null
+    private var listRoomExplore: MutableList<RoomExplore> = mutableListOf<RoomExplore>()
+
+    fun updateDataRoomExplore(context: Context, roomExplore: MutableList<RoomExplore>) {
+        this.context = context
+        this.listRoomExplore = roomExplore
```

Comment:
this có chức năng gì e?

Reply:
Em hiểu là nó tham chiếu tới biến instance của lớp hiện tại.